lib/server-jobs.sql

```diff
@@ -160,7 +167,7 @@ WHERE
         ) -- no running jobs and no recently finished jobs
     )
 RETURNING
-    js.*;
+    js.id;
```

Comment:
I suspect that in general, things will be slightly more performant if we're only returning columns we actually need?